sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -153,12 +154,16 @@ export class Recorder {
         if (isPlaybackMode()) {
           this.variables = rsp.bodyAsText ? JSON.parse(rsp.bodyAsText) : {};
         }
-        if (ensureExistence(this.sanitizer, "TestProxyHttpClient.sanitizer")) {
-          // Setting the recordingId in the sanitizer,
-          // the sanitizers added will take the recording id and only be part of the current test
-          this.sanitizer.setRecordingId(this.recordingId);
-          await handleEnvSetup(options.envSetupForPlayback, this.sanitizer);
-        }
+
+        // Setting the recordingId in the sanitizer,
+        // the sanitizers added will take the recording id and only be part of the current test
```

Comment:
// irrelevant comments

Update to something more meaningful? something that explains the next call?

Reply:
Ah yeah, forgot to drop these comments when I deleted the line below. Will just remove the comments. I think the code is reasonably self-explanatory.